packages/php/blueprint/tests/Unit/Exporters/ExportInstallPluginStepsTest.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Automattic\WooCommerce\Blueprint\Tests\Unit\Exporters;
+
+use Automattic\WooCommerce\Blueprint\Exporters\ExportInstallPluginSteps;
+use Automattic\WooCommerce\Blueprint\Tests\TestCase;
+
+class ExportInstallPluginStepsTest extends TestCase {
+
+	protected array $plugins = array(
+		'plugina/plugina.php' => array(
+			'Title' => 'plugina',
+			'RequiresPlugins' => array('pluginc'),
+		),
+		'pluginb/pluginb.php' => array(
+			'Title' => 'pluginb',
+			'RequiresPlugins' => array( 'plugina' ),
+		),
+		'pluginc/pluginc.php' => array(
+			'Title' => 'pluginc',
+			'RequiresPlugins' => array(),
+		),
+	);
+
+	private function get_mock() {
+		$mock = Mock( ExportInstallPluginSteps::class )->makePartial();
+		$mock->shouldReceive( 'wp_get_plugins' )->andReturn( $this->plugins );
+		return $mock;
+	}
+
+	/**
+	 * When everything is working as expected.
+	 *
+	 * @return void
+	 */
+	public function test_export() {
+	    $mock = $this->get_mock();
+		$mock->shouldReceive('wp_plugins_api')->andReturn((object) array(
+			'download_link' => 'download_link_url',
+		));
+
+		$result = $mock->export();
+		$this->assertCount( 3, $result );
```

Comment:
Do we need to test the result with the correct steps? Or just the number of steps is enough?

Reply:
@chihsuan Good call! Updated it in [Test each step from export](https://github.com/woocommerce/woocommerce/pull/54442/commits/444eb58484d0c6e71eb1c23201989ab99c0e8bc7)